dashboard/templates/guild/members.ejs

```diff
@@ -81,8 +81,8 @@
     $("table tbody").empty();
     for(var i = 0; i < members.length; i++) {
       member = members[i];
-      var color = member.roles.highest.hexColor === "#000000" ? '#FFFFFF' : member.roles.highest.hexColor;
-      var roles = member.roles.cache.filter(r=>r.name !== "@everyone").map(r=> {
+      var color = member.highestRole.hexColor === "#000000" ? '#FFFFFF' : member.highestRole.hexColor;

```

Comment:
Revert this change as `highestRole` no longer exists.

Reply:
It has no issues, [check this line](https://github.com/Fierythunder/guidebot/blob/878e2f04e65d5fea8c63711f73b9f268607a3f13/dashboard/dashboard.js#L329)

Reply:
Oh, I completely forgot that was a thing, I checked against the discord.js documentation and `highestRole` was no longer available. My apologies.

Reply:
in previous commit i also didn't see this line
so the members list didn't show up